### DISCUSSION TITLE
Using clappr video web player on Restreamer

### DISCUSSION THREAD

**Discussion by utilsites** [datarhei/restreamer#435] [Q&A] [answered]
On Restreamer-ui project source code, noticed that there are 2 video web players there: **videojs** and **clappr**.
I was expecting to have a setting where I could configure which player to use, although it seems that it's not the case. So the clappr is there from legacy and there is no easy way to use it?
(Need vast tags related plugin for clappr, already used and tested. Videojs may have the same but implies more time to implement)

**Comment by jstabenow** [comment, marked as answer]
A good hint, but it's not so easy:
Many video players + many users = many requirements.
That's why we need a concept that allows the integration of many video players. With one editor, the development becomes too complex and too error-prone. And that's why we didn't finish it.

Our idea:
Turn the video player into a publication service and make it copyable (like all other publication services). 

It is already on our roadmap. But without priority.
